src/thor/isochrone_action.cc

```diff
@@ -8,7 +8,7 @@ using namespace valhalla::midgard;
 namespace valhalla {
 namespace thor {
 
-std::string thor_worker_t::isochrones(Api& request) {
+std::string thor_worker_t::isochrones(Api& request, bool only_expand) {
```

Comment:
I thought it's nicer to have it as argument rather than implicitly skip when the `action` is `expansion`

Reply:
i actually prefer the other way around. here is my reasoning: i really want the top level apis to be as simple as possible and i want all of the info to process a request to be in the request object and not elsewhere. basically my main concern is about library api design.

Reply:
yeah, based on previous reviews I was literally 50:50 which way you'd prefer;) will change